Examples/Run/Reconstruction/TelescopeRecChi2Tracks.cpp

```diff
@@ -0,0 +1,15 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021-2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/TelescopeDetector/TelescopeDetector.hpp"
+
+#include "RecChi2Tracks.hpp"
+
+int main(int argc, char* argv[]) {
```

Comment:
general question about the example binaries: should those be removed in favor of python examples?

Reply:
I still need the binaries for the developing. But I already have put the removal on my list.